pkg/apis/operator/v1alpha1/doc.go

```diff
@@ -1,5 +1,5 @@
 // +k8s:deepcopy-gen=package
-// +groupName=ebs.aws.csi.openshift.io
+// +groupName=csi.openshift.io
 
 // Package v1alpha1 is the v1alpha1 version of the API.
 package v1alpha1 // import "github.com/aws-ebs-csi-driver-operator/pkg/apis/operator/v1alpha1"
```

Comment:
If you happen to need to add more changes to this PR, could you add *openshift* to this URL?

Reply:
fixed